src/test/test-copy.c

```diff
@@ -50,6 +51,135 @@ TEST(copy_file) {
         unlink(fn_copy);
 }
 
+TEST(copy_tree_replace_file) {
+        _cleanup_free_ char *buf = NULL;
+        char src[] = "/tmp/test-copy_file.XXXXXX";
+        char dest[] = "/tmp/test-copy_file.XXXXXX";
+
+        size_t sz = 0;
```

Comment:
No need to initialize `sz`, on a successful call to `read_full_file()` it will always be set.